images/build/Dockerfiles/Dockerfile

```diff
@@ -44,7 +42,7 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 # Check https://www.microsoft.com/net/platform/support-policy for support policy of .NET Core versions
-
+ENV NUGET_PACKAGES=/opt/nuget
```

Comment:
Target everything to go under `/opt` folder in this intermediate stage and in final image we copy back from `/opt/nuget` folder to `/var/nuget` folder and also set the environment variable appropriately.